spec/rubocop/cop/rspec/leaky_constant_declaration_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::LeakyConstantDeclaration do
+  subject(:cop) { described_class.new }
+
+  describe 'constant assignment' do
+    it 'flags inside an example group' do
+      bad_code = <<-RUBY
+        describe SomeClass do
+          CONSTANT = 'is not allowed'.freeze
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Stub constant instead of declaring explicitly.
+        end
+      RUBY
+      expect_offense(bad_code)
```

Comment:
Most of our specs are written as `expect_offense(<<-RUBY)`, i.e. not defining the code string as a local variable. I know we don’t exactly have a cop for it, but could we keep the consistency anyway? 😉